experimental/traffic-portal/src/app/core/profiles/profile-table/profile-table.component.ts

```diff
@@ -125,14 +137,16 @@ export class ProfileTableComponent implements OnInit {
 		private readonly route: ActivatedRoute,
 		private readonly navSvc: NavigationService,
 		private readonly dialog: MatDialog,
-		public readonly auth: CurrentUserService) {
+		public readonly auth: CurrentUserService,
+		private readonly exportService: ExportAttachmentService,
+		private readonly fileUtil: FileUtilsService) {
 		this.fuzzySubject = new BehaviorSubject<string>("");
 		this.profiles = this.api.getProfiles();
 		this.navSvc.headerTitle.next("Profiles");
 	}
 
 	/** Initializes table data, loading it from Traffic Ops. */
-	public ngOnInit(): void {
+	public async ngOnInit(): Promise<void> {
```

Comment:
why change this?

Reply:
it was a change while i was developing and forgot to revert that change back. Missed it :( got it corrected now.